job.go

```diff
@@ -6,10 +6,12 @@ import (
 	"time"
 )
 
+type jobInterval uint64
+
 // Job struct stores the information necessary to run a Job
 type Job struct {
-	sync.RWMutex
-	interval          uint64                   // pause interval * unit between runs
+	*sync.RWMutex
```

Comment:
Why pointer to the mutex?

Reply:
Oh, during development I've switched the Job struct to contain a sync.Locker, and the RWMutex Lock and Unlock functions have pointer receivers, so I had to do this. Then I've switched back to using a RWMutex and forgot to turn it back to value. I guess this is not a big deal (or am I missing something?) but I'll revert it nonetheless!